Makefile.am

```diff
@@ -34,6 +34,8 @@ else
 libyaksa_la_LDFLAGS = -version-info @libyaksa_so_version@
 endif !EMBEDDED_BUILD
 
+ze_native_TARGET = @enable_ze_native@
```

Comment:
Why is this sitting in the top-level Makefile?

Reply:
This seems to be the only am file there, variable ze_natibe_Target is an am variable. Adding to the ze/Makefile.mk did not work. But I may miss something. let me know if you have better solution.

Reply:
I'll need to experiment with it a bit to figure it out.  But this does not seem like the right solution.

Reply:
I looked at this a bit, and I'm not really sure what you are trying to do here.  Why are you defining `ze_native_TARGET` instead of directly using `@enable_ze_native@` inside `src/backend/ze/pup/Makefile.mk`?  The name `@enable_ze_native@` is a bit weird and probably should be changed, but functionality wise, this seems like a very round-about approach, unless I'm missing something here.

Reply:
fixed